TriggerRedHatBuild.groovy

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2020-present Sonatype, Inc. All rights reserved.
+ * Includes the third-party code listed at http://links.sonatype.com/products/clm/attributions.
+ * "Sonatype" is a trademark of Sonatype, Inc.
+ */
+@Grab('io.github.http-builder-ng:http-builder-ng-core:1.0.4')
+
+import groovyx.net.http.HttpBuilder
+import groovyx.net.http.HttpException
+
+if (args.size() < 3) {
+  println 'Usage: groovy TriggerRedhatBuild.groovy <version> <projectId> <apiKey>'
+  return
+}
+
+def (version, projectId, apiKey) = args
+
+final nextTag = getNextTag(apiKey, projectId, version)
+println "Deploying as ${nextTag}"
+
+build(apiKey, projectId, nextTag)
+
+final completedBuild = getCompletedBuild(apiKey, projectId, nextTag)
+
+println publish(apiKey, projectId, completedBuild.digest, completedBuild.name)
+
+String getNextTag(String apiKey, String projectId, String version) {
```

Comment:
javadoc please - I have no idea why we need a tag or what it means?

Reply:
👍 done.